vars/tdr.groovy

```diff
@@ -0,0 +1,11 @@
+def buildEndToEndTests(int delay, String buildJob, String[] params) {
+    build(job: buildJob, parameters: params, quietPeriod: delay, wait: false)
+}
+
+def setBuildStatus(String status, String desc) {
```

Comment:
It would be good to make it clear from the function name that this sets the build status in GitHub, or that it's some kind of external reporting function. Otherwise someone reading the function might assume it sets the _Jenkins_ build status.

Possibly `reportStatusToGitHub` or `reportBranchStatus`?